### DISCUSSION TITLE
Automatically fill in fields on create

### DISCUSSION THREAD

**Discussion by piegamesde** [go-shiori/shiori#538] [Feature requests]
New user here, I am a bit confused about how to get Shiori to automatically fill in fields. Currently my workflow is the following:

- Create a new bookmark, only fill in the URL, save
- "Update archive" that bookmark
- "Edit" the bookmark, read through the automatically added description, optionally adjust it, add tags
- "Refresh storage" to make sure that the UI displays the thumbnail

Instead, what I'd expect:

- Create a new bookmark, fill in the URL
- The remaining fields are filled in automatically, I can choose to edit them before saving
- On save, the bookmark gets archived
- Once there is a thumbnail, the view refreshes automatically

As a reference, this is how [Shaarly](https://github.com/shaarli/Shaarli) does it:

![image](https://user-images.githubusercontent.com/14054505/203180677-942670ba-fb4d-4804-9b81-ad9da1abcf77.png)
![image](https://user-images.githubusercontent.com/14054505/203180693-c105fd7d-2733-4b59-bfbd-187aba809a53.png)
![image](https://user-images.githubusercontent.com/14054505/203180711-4928a74a-a223-479b-b161-dfcd67213da6.png)

I think that's a very nice of interacting, and I'd like to see something similar in Shiori

**Comment by fmartingr** [comment]
Hey @piegamesde, if you could record a video it would be awesome! Right now, fields are automatically retrieved from the URL (when possible), the problem is that you may not see that until you refresh the page after a few seconds, but title/description/image/etc should be retrieved automatically. 

I'm not very frontend-savvy so I haven't made many improvements over that side of Shiori.

**Reply by ndegruchy** [reply]
Hey, so I'm having the exact same issue. I updated to 1.55 and it continues to persist. If I hit "update archive", *then* it fetches the title. As requested (though I'm not the OP), attached is a video.

[2023-08-07 09-24-10.webm](https://github.com/go-shiori/shiori/assets/52262673/4ae9c654-41a0-4e89-af7a-58d89ab3ea3a)

**Reply by fmartingr** [reply]
Hey @ndegruchy! Thanks for the video. We made a bunch of fixes that are not currently released but fix this particular issue (documented in #654). It will be released with 1.6.0.

If 1.6.0 takes too long to release, I could backport the fixes and release 1.5.6.

**Reply by ndegruchy** [reply]
Thanks for responding. I can wait. The work around isn't too onerous and otherwise the app works fine.
